resources/metadata/mtaBuild.yaml

```diff
@@ -224,6 +224,7 @@ spec:
         type: piperEnvironment
         params:
           - name: mtarFilePath
+          - name: mtaBuildToolDesc
```

Comment:
this should be a custom field, otherwise it cannot be handled properly on `groovy` side I guess